src/components/Project/components/Rewards.tsx

```diff
@@ -2,15 +2,23 @@ import { useJbProject } from '@/hooks/useJbProject'
 import Image from 'next/image'
 import { twMerge } from 'tailwind-merge'
 import { RewardCard, RewardCardSkeleton } from './RewardCard'
+import { useEffect, useMemo, useState } from 'react'
 
 export type RewardsProps = {
   className?: string
 }
 
 export const Rewards: React.FC<RewardsProps> = ({ className }) => {
   const { nftData } = useJbProject()
+  const [initialNftDataLoaded, setInitialNftDataLoaded] = useState(false)
 
-  if (nftData.isLoading) {
+  useEffect(() => {
```

Comment:
eh, wonder if theres a better way to implement the sdk hook. this works for now tho